js/lib/ui.js

```diff
@@ -467,6 +442,5 @@ elgg.ui.initAccessInputs = function () {
 };
 
 elgg.register_hook_handler('init', 'system', elgg.ui.init);
-elgg.register_hook_handler('init', 'system', elgg.ui.initDatePicker);
```

Comment:
This would stop the auto-wiring of .elgg-input-date inputs not using the input/date view. For BC we really have to catch them all.